File: Programmers/Lv2_주식가격.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution2(vector<int> prices) {
	vector<int> answer;
	int count;

	for (int i = 0; i < prices.size() - 1; i++) {
		count = 0;
		for (int j = i + 1; j < prices.size(); j++) {
			count++;
			if (prices[i] > prices[j])
				break;
		}
		answer.push_back(count);
	}
	answer.push_back(0);
	return answer;
}

File: Programmers/Lv2/Lv2_주식가격.py
def solution(prices):
    answer = []

    for i in range(0, len(prices) - 1):
        count = 0
        for j in range(i + 1, len(prices)):
            count += 1
            if prices[i] > prices[j]:
                break
        answer.append(count)
    answer.append(0)
    return answer